### DISCUSSION TITLE
Table Filter Grouping/Section

### DISCUSSION THREAD

**Discussion by kuchienkz** [filamentphp/filament#8172] [Help] [answered]
Is there a way to Group some table Filters and add a heading into it, just like Section in forms?

Currently, I have 12 (and possibly more ) Filters. I think it will look better with groups/sections.

Im using filament 2.x btw.

**Comment by leandrocfe** [comment, marked as answer]
you can use with [custom-filter-forms](https://filamentphp.com/docs/2.x/tables/filters#custom-filter-forms)

```php
->filters([
    Filter::make('my_filter')
        ->form([
            Section::make('Group 1')
                ->schema([
                    Select::make('filter_a'),
                    Select::make('filter_b')
                    ...
                ])
            ...
        ])
        ...
])
```

**Comment by kuchienkz** [comment]
While we are at it, is there a way to get current Filter values/states? 

I made a "Export to Excel" function. I want to export rows based on current Filter states.

**Reply by danharrin** [reply]
`$this->tableFilters` or similar
